### PR TITLE
incarcerated user can connect to voice call

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**

### CODE REVIEW

File: src/components/Modals/VoiceCallModal.tsx [hunk 1]
```diff
@@ -22,18 +27,63 @@ const VoiceCallModal = ({ data, onCloseModal }: Props) => {
   //   TODO: update state of isConnected after the room client is created successfully
   const [isConnected, setIsConnected] = useState(false);
   const call = data.entity;
+  const { authInfo } = useAppSelector((state) => state.session);
+  const [playJoinCall] = useSound(JoinCallSound);
+  const [playLeaveCall] = useSound(LeaveCallSound);
+
+  if (!call.call?.url || !call.call.token) throw new Error("Invalid call data");
+
+  const handlePeerConnected = useCallback(() => {
```
Comment (gabrielsaruhashi) [inline src/components/Modals/VoiceCallModal.tsx:36]: a lot of logic is dupped from the video call component but I'm pretty certain the implementation will change significantly very soon

File: src/components/Call/Call.tsx [hunk 2]
```diff
@@ -123,6 +123,7 @@ const CallScreen: React.FC<Props> = React.memo(
         id: call.id,
         url: call.call.url!,
         token: call.call.token!,
+        type: CallType.VOICE_CALL,
```
Comment (cainlevy) [inline src/components/Call/Call.tsx:126]: is this the video call component?

Comment (gabrielsaruhashi) [reply src/components/Call/Call.tsx:126]: nope currently using the <Audio/>
component

Comment (gabrielsaruhashi) [reply src/components/Call/Call.tsx:126]: yep!

**Comment by gabrielsaruhashi** [comment]
@cainlevy i havent been able to get local calls working yet to test this out fully, but as you are out today, it might be helpful for u to take a peek already so I can keep building on top of this work if it looks like a good starting point

File: src/components/Call/Call.tsx (hunk 2 above)
Comment (cainlevy) [reply src/components/Call/Call.tsx:126]: should this be

```suggestion
        type: CallType.VIDEO_CALL,
```

**Comment by cainlevy** [comment]
i peeked! just want to ensure continuity for video calls.

File: src/components/Call/Call.tsx (hunk 2 above)
Comment (gabrielsaruhashi) [reply src/components/Call/Call.tsx:126]: oh lordy 🤦 u prevented a catastrophe

**Review by cainlevy — APPROVED** [review]

File: src/components/Modals/VoiceCallModal.tsx (hunk 1 above)
Comment (cainlevy) [reply src/components/Modals/VoiceCallModal.tsx:36]: i'm 👍 on forking the video call components and then extracting the shared pieces later. composition is much easier to maintain in react than intertangled conditionals.

**Pull request merged**